pyaerocom/helpers.py

```diff
@@ -227,13 +233,18 @@ def make_dummy_cube_latlon(lat_res_deg=2, lon_res_deg=3, lat_range=None, lon_ran
     Cube
         dummy cube in input resolution
     """
+
     if lat_range is None:
         lat_range = (-90, 90)
     if lon_range is None:
         lon_range = (-180, 180)
 
-    lons = np.arange(lon_range[0] + lon_res_deg / 2, lon_range[1] + lon_res_deg / 2, lon_res_deg)
-    lats = np.arange(lat_range[0] + lat_res_deg / 2, lat_range[1] + lat_res_deg / 2, lat_res_deg)
+    lons = np.arange(
+        lon_range[0] + (lon_res_deg / 2), lon_range[1] + (lon_res_deg / 2), lon_res_deg
+    )
+    lats = np.arange(
+        lat_range[0] + (lat_res_deg / 2), lat_range[1] + (lat_res_deg / 2), lat_res_deg
+    )
```

Comment:
why the extra parenthesis?
``` python
>>> 3+1/2
3.5
```

Reply:
I visually find it more explicit, but this is personal preference

Reply:
Respectfully, I disagree ;) I think my change is the best balance between readability and efficiency. The way I have it, the endpoints of the interval are contained within a single positional argument, separated by commas. By inspecting this argument, you can see that the interval start and end are composed of two pieces: the bounds of the range provided in `lon_range` or `lat_range`, plus an offset contained in parenthesis. To me, this is the most natural way of thinking about it and cleanest way of representing that, but again, this probably boils down to personal preference and efficiency is probably the more important.

On to something else, would you mind taking a look at why CI is failing? I think it's happening on the installation. Even the linters are failing because `ImportError('NumPy 1.10+ is required to install cartopy.')`

Reply:
> Respectfully, I disagree ;) [...] To me, this is the most natural way of thinking about it and cleanest way of representing that, but again, this probably boils down to personal preference and efficiency is probably the more important.

I agree your way is better.

> On to something else, would you mind taking a look at why CI is failing? I think it's happening on the installation. Even the linters are failing because `ImportError('NumPy 1.10+ is required to install cartopy.')`

I noticed the CI failing. Not only the docs and lint jobs are failing, but the Python 3.8 venv is failing one of the tests. Will take a look at it.

Reply:
Thanks for taking a look. It is strange because I don't think I changed anything that would effect other tests. I see more than one different test failing for different runs

Reply:
#686 should take care of fixing the CI install errors.
You can marge `main-dev` into this branch after #686 is merged.